CHANGES-5.0.md

```diff
@@ -10,6 +10,7 @@
 * Speed up updating the configuration, When some nodes in the cluster are down. [#8857](https://github.com/emqx/emqx/pull/8857)
 * Fix delayed publish inaccurate caused by os time change. [#8926](https://github.com/emqx/emqx/pull/8926)
 * Fix that EMQX can't start when the retainer is disabled [#8911](https://github.com/emqx/emqx/pull/8911)
+* Fix empty variable interpolation in authentication and authorization [#8963](https://github.com/emqx/emqx/pull/8963)
```

Comment:
Suggest that this change should be put into 5.0.9? Because 5.0.8 is about to be frozen and ready for release

Reply:
✅